src/lib/clipboard-access.js

```diff
@@ -1,24 +1,31 @@
 /**
+ * writeUsingTextarea writes text to clipboard by injecting an input box on the web page.
  *
  * @param {string} text The text to be copied
- * @return {Promise} contains original response
+ * @return {Promise}
  */
-export default async function copy(text) {
-  try {
-    // For Firefox. Use native API. This won't work on Chrome since it requires focused document.
-    // See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard
-    await navigator.clipboard.writeText(text);
-    return true;
-  } catch (error) {
-    // For Chrome. Use an input box in background page to interact with clipboard.
-    /** @type {HTMLTextAreaElement} */
-    const textBox = document.createElement('textarea');
-    document.body.append(textBox);
-    textBox.value = text;
-    textBox.select();
-    document.execCommand('Copy');
-    textBox.value = '';
-    document.removeChild(textBox);
-    return true;
-  }
+function writeUsingTextarea(text) {
+  /** @type {HTMLTextAreaElement} */
+  const textBox = document.createElement('textarea');
+  document.body.append(textBox);
+  textBox.value = text;
+  textBox.select();
+  document.execCommand('Copy');
+  document.body.removeChild(textBox);
```

Comment:
Firefox doesn't like people using `document.execCommand` (they prefer `navigator.clipboard`) but this is the only way that an extension on Clipboard can write to clipboard from background script.